openfpga_flow/openfpga_timing_annotation/k6_frac_N10_tileable_adder_chain_dpram1K_fracff_skywater130nm_tt.yml

```diff
@@ -0,0 +1,47 @@
+INPAD_DELAY: 0.11e-9

```

Comment:
These timing values are extracted from a k4 architecture at skywater 130nm. 
In a k6 arch, some timing values should be changed. 
Have you done any scaling from k4 to k6? If not, I am o.k. but you need to add comments in this yml file.

Reply:
Yes, all of the lines that have the comment at the end of them saying the timing is innacurate are all from 40nm. I used 130nm timing values for the non-commented lines, though.